source/includes/php/rest_client.md

```diff
@@ -7,10 +7,10 @@
 require_once 'path/to/lib/shoprunback-php/init.php';
 
 $restClient = \Shoprunback\RestClient::getClient();
-$restClient->setApiBaseUrl('https://dashboard.shoprunback.com');
+$restClient->setApiBaseUrl('http://localhost:3000');
```

Comment:
no, you are the only developer who may use localhost as API base URL, others will use the sandbox/production.